README.md

```diff
@@ -1,37 +1,117 @@
 
 # customs-financials-email-throttler
 
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) [![Coverage](https://img.shields.io/badge/test_coverage-90-green.svg)](/target/scala-3.3.5/scoverage-report/index.html) [![Accessibility](https://img.shields.io/badge/WCAG2.2-AA-purple.svg)](https://www.gov.uk/service-manual/helping-people-to-use-your-service/understanding-wcag)
+
 A backend service to control the rate at which emails are dispatched.
 
 This application lives in the "public" zone.
 
-## Running the application locally
+## Running the service
+
+*From the root directory*
+
+`sbt run` - starts the service locally
+
+`sbt runAllChecks` - Will run all checks required for a successful build
+
+Default service port on local - 9872
+
+### Required dependencies
 
-The service has the following dependencies:
+There are a number of dependencies required to run the service.
+
+The easiest way to get started with these is via the service manager CLI - you can find the installation guide [here](https://docs.tax.service.gov.uk/mdtp-handbook/documentation/developer-set-up/set-up-service-manager.html)
+
+| Command                                         | Description                                                      |
+|-------------------------------------------------|------------------------------------------------------------------|
+| `sm2 --start CUSTOMS_FINANCIALS_ALL`            | Runs all dependencies                                            |
+| `sm2 --start CUSTOMS_EMAIL_FRONTEND_ALL `       | Runs all email dependencies                                      |
+| `sm2 -s`                                        | Shows running services                                           |
+| `sm2 --stop CUSTOMS_FINANCIALS_EMAIL_THROTTLER` | Stop the micro service                                           |
+| `sbt run` or `sbt "run 9872"`                   | (from root dir) to compile the current service with your changes |
+
+
+### Runtime Dependencies
+(These are subject to change and may not include every dependency)
 
 * `AUTH`
 * `AUTH_LOGIN_STUB`
 * `AUTH_LOGIN_API`
+* `BAS_GATEWAY`
+* `CA_FRONTEND`
+* `SSO`
 * `USER_DETAILS`
 * `EMAIL`
 * `CONTACT_FRONTEND`
 
-You can use the CUSTOMS_EMAIL_THROTTLER profile in service manager to start these services.
+### Login enrolments
```

Comment:
Better to remove this section entirely as its a backend service and seems no endpoint requires the Enrolments